kong/router/atc.lua

```diff
@@ -62,6 +62,7 @@ do
     ["String"] = {"net.protocol", "tls.sni",
                   "http.method", "http.host",
                   "http.path",
+                  "http.path.segments.*",
```

Comment:
Do we need any additional code to reject invalid inputs for this field (e.g. `http.path.segments.abc` or `http.path.segments.4_1` or `http.path.segments.1_2_3`)? I'm not sure if other fields are validated or not so maybe this is not something we consider a hard requirement for the expression router syntax, but it seems like it would be nice to have.

Reply:
Good idea, we should be able to validate the field name with `get_fields()` and reject invalid ones at schema validation time.

Reply:
done.